ui/src/app/views/recent-work/component.css

```diff
@@ -16,15 +18,14 @@
 }
 
 .scroll-indicator {
-  cursor: pointer;
```

Comment:
My `cursor: pointer;` change seems to have been reverted. Was this intentional or just a bad merge?

Reply:
probably a bad merge.  i will fix with 1486